### ISSUE TITLE
Ferdium won't launch after upgrading Ubuntu to 24.04 (using AppImage)

### ISSUE THREAD

**Issue by leoheck** [ferdium/ferdium-app#1694]
### Your issue

Ferdium is not opening anymore after upgrading ubuntu from 23.10 to the unreleased 24.04.

This is what I get. Even the help to get ferdium info is not available anymore.

![image](https://github.com/ferdium/ferdium-app/assets/1277920/90c7d099-b537-4e82-8373-6b46b7312e33)

However this is the AppImage I am using.
```
Ferdium-linux-Portable-6.7.3-nightly.11-x86_64_4a1b12d6d452e18c9051545269e61317.AppImage
```

Launching it from the command line, this is the message I got.
```bash
➜ ./Ferdium-linux-Portable-6.7.3-nightly.11-x86_64_4a1b12d6d452e18c9051545269e61317.AppImage 
QSocketNotifier: Can only be used with threads started with QThread
[14565:0416/153718.391657:FATAL:setuid_sandbox_host.cc(158)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_FerdiuAUvtnU/chrome-sandbox is owned by root and has mode 4755.
```

OS version
```
~/Applications …
➜ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu Noble Numbat (development branch)
Release:	24.04
Codename:	noble
```

**Comment by leoheck** [comment]
Weird that this is the latest version of Ferdi I see on the Releases page `6.7.3-nightly.9` that was released 3 days ago but I have this `6.7.3-nightly.11` instead.

```
[Ferdium-linux-Portable-6.7.3-nightly.9-x86_64.AppImage](https://github.com/ferdium/ferdium-app/releases/download/v6.7.3-nightly.9/Ferdium-linux-Portable-6.7.3-nightly.9-x86_64.AppImage)
```

**Comment by leoheck** [comment]
Same issue happens with this verison `Ferdium-linux-Portable-6.7.3-nightly.9-x86_64_cdca802a7cee1d127f07bebdefeb5ef2.AppImage`

**Comment by SpecialAro** [comment]
> Weird that this is the latest version of Ferdi I see on the Releases page `6.7.3-nightly.9` that was released 3 days ago but I have this `6.7.3-nightly.11` instead.
> 
> ```
> [Ferdium-linux-Portable-6.7.3-nightly.9-x86_64.AppImage](https://github.com/ferdium/ferdium-app/releases/download/v6.7.3-nightly.9/Ferdium-linux-Portable-6.7.3-nightly.9-x86_64.AppImage)
> ```


Version 6.7.3-nigthly.11 was released today and had a bug that is reported here: https://github.com/ferdium/ferdium-app/issues/1689

Unfortunately I think you made the Ubuntu upgrade and, at the same time, the Ferdium upgrade... Which caused the black screen you are seeing.

It is weird that nightly.9 is not working tho...

**Comment by leoheck** [comment]
Hmm, interesting, now I am not sure the issue is related.

**Comment by mcmxcdev** [comment]
It might also be related to https://github.com/ferdium/ferdium-app/issues/1535 ?

I am not using Ubuntu 24.04 yet (probably not gonna upgrade for a month still), so it would be highly appreciated to be debugged by the community :)

**Comment by SpecialAro** [comment]
@leoheck can you check if you can upgrade to the latest nightly and if this issue is fixed?

**Comment by leoheck** [comment]
Sure thing. I am going to come back after a few minutes.

**Comment by leoheck** [comment]
Yeah the version 
```
Version: 6.7.3-nightly.12
Server: Ferdium Server
Electron: 30.0.0
Chrome: 124.0.6367.49
Node.js: 20.11.1
Platform: Linux 6.8
Arch: x64
Build date: Wed Apr 17 2024 09:24:46 GMT-0300 (Brasilia Standard Time)
Git SHA: deba94e5
Git branch: nightly
```

Works again, actually while downloading it, version `6.7.3-nightly.9` started to work... maybe ubuntu upgraded something.

**Comment by SpecialAro** [comment]
Great 😃 glad it's fixed!

**Comment by leoheck** [comment]
@SpecialAro the new `nightly.13` is not working anymore.

![image](https://github.com/ferdium/ferdium-app/assets/1277920/5e1349de-87bb-4ddd-a501-6a70ec0b1b9b)

**Comment by leoheck** [comment]
@SpecialAro, could you reopen this issue? I cannot see the button for that.

**Comment by SpecialAro** [comment]
I can reopen, but I don't think it is related to the same issue. I see that you can not open the app at all

**Comment by leoheck** [comment]
That's true when launching from the command line. When using the launcher it shows the icon on tray.

**Comment by leoheck** [comment]
However the issue here is that it is not launching. So I think it is still related to the report.

**Comment by mcmxcdev** [comment]
Crossreferencing https://github.com/laurent22/joplin/issues/10332 and https://github.com/logseq/logseq/issues/11240 here which seem to be suffering from the same issue.

**Comment by Alphrag** [comment]
@leoheck Do you know if the problem is also happening when installing Ferdium through the `.deb` file? I am also on Ubuntu 23.10 at the moment, and might update soon to 24.04, and we will need to see if we can provide a patch for that before those jumping from the previous LTS get this problem.

**Comment by leoheck** [comment]
Hey @Alphrag , I am on Ubuntu 24.04 (not a clean install) it was upgraded when it was still to be released. It was released yesterday. However, it has been updating a bunch of things daily since when I upgraded it. It may have eventually improved/fixed something.

Now, talking about the Ferdium, it is currently working. There are no problems anymore.
But I am not sure about the .deb version of it. 

I am running this version
```
Version: 6.7.3-nightly.15
Server: Ferdium Server
Electron: 30.0.1
Chrome: 124.0.6367.60
Node.js: 20.11.1
Platform: Linux 6.8
Arch: x64
Build date: Sat Apr 20 2024 21:50:37 GMT-0300 (Brasilia Standard Time)
Git SHA: fe61e6ba
Git branch: nightly
```
